hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/type/UdtObjectToJsonFunctionTest.java

```diff
@@ -46,7 +46,7 @@ public class UdtObjectToJsonFunctionTest extends SqlJsonTestSupport {
     @BeforeClass
     public static void beforeClass() {
         Config config = smallInstanceConfig()
-                .setProperty(SQL_CUSTOM_TYPES_ENABLED.getName(), "true");
+                .setProperty(ClusterProperty.SQL_CUSTOM_CYCLIC_TYPES_ENABLED.getName(), "true");
```

Comment:
flag not needed by most (all?) tests

Reply:
It is needed only for one test, but it makes sense to leave it as it is: this test is a unit test for corresponded function.